src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/SemanticTokensRangeTests.cs

```diff
@@ -28,41 +30,36 @@ static class C { }
             var range = new LSP.Range { Start = new Position(0, 0), End = new Position(2, 0) };
             var results = await RunGetSemanticTokensRangeAsync(testLspServer, testLspServer.GetLocations("caret").First(), range);
 
+            // Everything is colorized syntactically, so we shouldn't be returning any semantic results.
             var expectedResults = new LSP.SemanticTokens
             {
-                Data = new int[]
-                {
-                    // Line | Char | Len | Token type                                                               | Modifier
-                       0,     0,     10,   SemanticTokensHelpers.TokenTypeToIndex[LSP.SemanticTokenTypes.Comment],      0, // '// Comment'
-                       1,     0,     6,    SemanticTokensHelpers.TokenTypeToIndex[LSP.SemanticTokenTypes.Keyword],      0, // 'static'
-                       0,     7,     5,    SemanticTokensHelpers.TokenTypeToIndex[LSP.SemanticTokenTypes.Keyword],      0, // 'class'
-                       0,     6,     1,    SemanticTokensHelpers.TokenTypeToIndex[ClassificationTypeNames.ClassName],   (int)TokenModifiers.Static, // 'C'
-                       0,     2,     1,    SemanticTokensHelpers.TokenTypeToIndex[ClassificationTypeNames.Punctuation], 0, // '{'
-                       0,     2,     1,    SemanticTokensHelpers.TokenTypeToIndex[ClassificationTypeNames.Punctuation], 0, // '}'
-                },
+                Data = Array.Empty<int>()
             };
 
-            await VerifyNoMultiLineTokens(testLspServer, results.Data!).ConfigureAwait(false);
             Assert.Equal(expectedResults.Data, results.Data);
         }
 
         [Fact]
-        public async Task TestGetSemanticTokensRange_PartialDocAsync()
```

Comment:
did this test go missing?  I'm not seeing a new test that verifies the partial (1,0) -> (2,0) positions

Reply:
Whoops, you're totally right. Thanks! Added it back.